module/proto/gravity/v1/genesis.proto

```diff
@@ -99,6 +99,10 @@ message Params {
     (gogoproto.nullable)   = false
   ];
   uint64 unbond_slashing_valsets_window = 17;
+  bytes slash_fraction_bad_eth_signature = 18 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
```

Comment:
this needs to be a string. Also, what does a "bad" signature mean in this context

Reply:
a signature over some batch, validator set, or logic call not already in the store.